### PR TITLE
Upgrade SciLab to v6.0.2

### PR DESCRIPTION
**Pull request by ghost**
http://help.scilab.org/docs/6.0.2/en_US/CHANGES.html

### CODE REVIEW

**Comment by cixtor** [comment]
How many times are you going to create this?

• https://github.com/Homebrew/homebrew-cask/pull/60545  
• https://github.com/Homebrew/homebrew-cask/pull/60546  
• https://github.com/Homebrew/homebrew-cask/pull/60547  
• https://github.com/Homebrew/homebrew-cask/pull/60549

File: Casks/scilab.rb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  appcast 'https://www.scilab.org/'
   name 'Scilab'
   homepage 'https://www.scilab.org/'
 
+  depends_on macos: '>= :sierra'
+
   app "scilab-#{version}.app"
+
+  caveats do
+    depends_on_java '8'
```
Comment (vitorgalvao) [inline Casks/scilab.rb]: Are you sure? The page seems to suggest it requires version 6 (the one provided by Apple).

Comment (sayomelu) [reply Casks/scilab.rb]: Yes, it is 6. It is a mistake.

**Review by ghost — COMMENTED** [review]
Fix depends

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**